apps/webapp/src/Index.tsx

```diff
@@ -1,32 +1,57 @@
-import { useState } from "react"
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+import gavel from './assets/gavel.jpg'
+import socialMediaIcons from './assets/social-media-icons.jpg';
 
 export default function Index() {
-  const [count, setCount] = useState(0)
 
   return (
     <>
-      <div>
-        <a href="https://vite.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
+      <nav className="max-w-md flex-responsive nav-row">
+        <div>
+          <img src={gavel} alt="gavel icon" className="gavel-icon" />
+          <strong className="text-lg">Trust Assembly</strong>
+        </div>
+        <div className="flex-grow"></div>
+        <ul className="nav-list flex-row gap">
+          <li>Home</li>
+          <li>Explore</li>
+          <li>About</li>
+          <li className="button-link">Login</li>
```

Comment:
wrap each of these in a `Link` component (from `react-router-dom`). The link can go to `#` for now